webpack/assets/javascripts/react_app/components/common/forms/NumericInput.js

```diff
@@ -1,48 +1,48 @@
-import NumericInput from 'react-numeric-input';
+import ReactNumericInput from 'react-numeric-input';
 import React from 'react';
 import PropTypes from 'prop-types';
 
 import { noop } from '../../../common/helpers';
 import CommonForm from './CommonForm';
 
-const TextInput = ({
+const NumericInput = ({
   label,
   className,
   value,
-  onChange,
+  onValueChange,
```

Comment:
this will block others who would like to use this component with regular `onChange`

Reply:
Yeah, but that is the basic problem. This component actually do not support `onChange`